requirements.txt

```diff
@@ -1,4 +1,4 @@
 Flask
 Flask-PyMongo
 Flask-WTF
-in-toto==0.1.1
\ No newline at end of file
+-e git://github.com/in-toto/in-toto.git@v0.2.3.dev5#egg=in-totov0.2.3.dev5
```

Comment:
We should bump this. Worth using a dependency bot here to bump the versions automagically:tm:?

Reply:
```suggestion
in-toto
```